<launch>
	<node
	pkg="proportional_controller"
	type="odom_control_obstacle_v1"
	name="Odometry_Control"
	required="true"
	launch-prefix="xterm -e"
    />

  <node
	pkg="proportional_controller"
	type="yaw_control_service"
	name="turn_service"
	required="true"
	launch-prefix="xterm -e"
	/>

<include file="$(find turtlebot3_bringup)/launch/turtlebot3_remote.launch">
  </include>

 <!-- <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch">
  </include>-->
  
<!--	<include file="$(find turtlebot3_teleop)/launch/turtlebot3_teleop_key.launch">
  </include>-->


</launch>
